src/actions/timeEntries.js

```diff
@@ -15,13 +15,13 @@ export const getTimeEntriesForWeek = (userId, offset) => {
     .tz('America/Los_Angeles')
     .startOf('week')
     .subtract(offset, 'weeks')
-    .format('YYYY-MM-DD');
+    .format('YYYY-MM-DDTHH:mm:ss');
```

Comment:
Just curious, what is the point of adding the hours/minutes/seconds? We don't see these being displayed in the time logs (as expected), so I was wondering what purpose it served.